test/pooling.jl

```diff
@@ -813,15 +813,15 @@ end
 @testset "AutoDiff: spatial_rank=$spatial_rank" for spatial_rank in (1, 2)
   x = rand(rng, repeat([10], spatial_rank)..., 3, 2)
   pdims = PoolDims(x, 2)
-  gradtest(x -> maxpool(x, pdims), x; broken=spatial_rank <= 2)
+  gradtest(x -> maxpool(x, pdims), x, broken=spatial_rank <= 0) # was <= 2 before
```

Comment:
Is this what you meant? I think it's still messed up.

Reply:
that's fine. This should be properly fixed somewhere else, but we need to have tests passing in this PR